zerver/lib/user_groups.py

```diff
@@ -316,7 +324,18 @@ def create_system_user_groups_for_realm(realm: Realm) -> Dict[int, UserGroup]:
         everyone_on_internet_system_group,
     ]
 
+    creation_time = timezone_now()
     UserGroup.objects.bulk_create(system_user_groups_list)
+    RealmAuditLog.objects.bulk_create(
+        RealmAuditLog(
+            realm=realm,
+            acting_user=None,
+            event_type=RealmAuditLog.USER_GROUP_CREATED,
+            event_time=creation_time,
+            modified_user_group=user_group,
+        )
+        for user_group in system_user_groups_list
+    )
```

Comment:
Does a comment belong here that we don't create membership audit log entries for system groups, because those membership change are already tracked in the actual user tables?

Reply:
Yeah, I think for system groups, this function itself doesn't add users to them. But I think we do need to track that for user creation and role changes.

Reply:
Yeah, the behavior is correct -- I just thing we should document it in a code comment, not just in the commit message.